lib/modules/schedule/widget/loading_screen_widget/loading_screen_widget.dart

```diff
@@ -0,0 +1,126 @@
+                        Icons.keyboard_arrow_left,
+                        color: CommonColors.green,
+                      )),
+                ),
+                Expanded(
+                  child: IconButton(
+                      onPressed: () {},
+                      icon: Icon(
+                        Icons.keyboard_arrow_right,
+                        color: CommonColors.green,
+                      )),
+                )
+              ],
+            ),
+          ),
+        ),
+        const Divider(
+          thickness: 1.0,
+        ),
+        Expanded(
+            flex: 2,
+            child: Container(
+              padding: EdgeInsets.all(16),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                      ShimmerLoading(isLoading: true, child: AbbrBlock()),
+                    ],
+                  ),
+                  ShimmerLoading(
+                    isLoading: true,
+                    child: Container(
+                        height: 48,
+                        width: 343,
+                        decoration: BoxDecoration(
+                          color: CommonColors.neutral,
+                          borderRadius: BorderRadius.circular(16),
+                        )),
+                  ),
+                ],
+              ),
+            )),
+        Container(
+          padding: EdgeInsets.all(16),
+          alignment: Alignment.topLeft,
+          child: Text(
+            "SCHEDULE FOR THE WEEK",
+            style: TextStyle(color: CommonColors.neutral[200]),
+          ),
+        ),
+        Expanded(
+            flex: 9,
+            child: ListView.separated(
+              itemBuilder: (context, index) => ShimmerLoading(
+                isLoading: true,
+                child: ScheduleItemShimmer(
+                  schedule: ScheduleMockedEntities.mockWeek[index],
+                ),
+              ),
+              separatorBuilder: (context, index) => const Divider(
+                indent: 70,
+                endIndent: 18,
+                thickness: 1,
+              ),
+              itemCount: ScheduleMockedEntities.mockWeek.length,
+              shrinkWrap: true,
+            ))
+      ],
+    );
+  }
+
+  Widget AbbrBlock() => Container(
```

Comment:
Always write in camel case format. (lowerCamelCase)
Always write function or method name with these formats
    - verb followed by a noun.
    - preposition followed by a noun then a verb.